index.js

```diff
@@ -1,35 +1,65 @@
-/*!
- * forwarded
- * Copyright(c) 2014 Douglas Christopher Wilson
- * MIT Licensed
```

Comment:
We want to keep these headers so we know where in the source the copyrights lives (and so the note is retained when people copy the file but not the license).